Casamari/MNC001.xml

```diff
@@ -844,15 +844,18 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </item>
                         <item xml:id="e2">
                            <desc>
-                              Extremely frequent corrections, textual additions, glosses or numbers of the subsection, nearly on every page in the margins, in the intercolumn or interlineally.
-                              They are written in several hands and resulting from collation with one or more additional exemplars.
+                              Extremely frequent corrections, textual additions, glosses or numbers of the subsection, nearly on every leaf in the margins, in the intercolumn or interlineally.
+                              They are written in several hands and probably result from collation with one or more additional exemplars.
                               Sometimes the additions in the margins are encircled in rectangular frames and are most often called in the body of the text by <foreign xml:lang="gez">tamallas</foreign>-signs in the shape of a cross or two perpendicular lines (e.g. <locus target="#6vc #19rb #30rb"/>).
                               Sometimes they are overwritten on erasure (e.g. on <locus target="#4rb #7va #24vb"/>), especially towards the second part of the manuscript.
                            </desc>
+                           <!-- Insert one or two selected examples to be printed in our catalogue!
+                           SH-MV 2021-01-13
+                           -->
                         </item>
                         <item xml:id="e3">
                            <desc>
-                              The titles of the some of the Old Testament books are written in the upper margin in black or red ink and marked with thin lines above and below, 
+                              Titles of the some of the Old Testament books are written in the upper margin in black or red ink and marked with thin lines above and below, 
```

Comment:
Sorry, I confused you; I meant:
The titles of some of the Old Testament books

The second "the" should be deleted, or not?

Reply:
Yes, of course, you are right! 
sorry!